pad_api_data/pad_etl/processor/enemy_skillset.py

```diff
@@ -656,8 +370,7 @@ def __init__(self, skill: EnemySkillRef):
         self.max_turns = params(skill)[2]
         super().__init__(
             skill,
-            effect='skill_bind',
-            description=Describe.bind(self.min_turns, self.max_turns, target_type='active skills')
+            description=Describe_EN.bind(self.min_turns, self.max_turns, targets=Describe_EN.ACTIVE)
```

Comment:
Not something you have to do now if you don't want, but I think probably the right way to handle this is to to do

```
@property 
def description():
  return Describe_EN.bind(...
```

Reply:
Actually thinking about how I'm going to use these later, I'll want to call Describe_JP.xyz() on each translated skill.

It doesn't really make sense to duplicate this logic everywhere, the 'description' function should probably take a Describe as an input, with a default of Describe_EN.